src/commonMain/kotlin/au/id/micolous/metrodroid/transit/en1545/En1545TransitData.kt

```diff
@@ -36,25 +37,34 @@ abstract class En1545TransitData : TransitData {
             if (mTicketEnvParsed.contains(ENV_NETWORK_ID))
                 li.add(ListItem(R.string.en1545_network_id,
                         mTicketEnvParsed.getIntOrZero(ENV_NETWORK_ID).toString(16)))
-            mTicketEnvParsed.getTimeStampString(ENV_APPLICATION_VALIDITY_END, tz)?.let {
-                li.add(ListItem(R.string.expiry_date, it))
+
+            mTicketEnvParsed.getTimeStamp(ENV_APPLICATION_VALIDITY_END, tz)?.let {
+                li.add(ListItem(R.string.expiry_date, TimestampFormatter.longDateFormat(it)))
```

Comment:
Expiry date may contain time as well. format() is safer. Same for several calls below

Reply:
Fixed in 70d4d42f